docs/integrations.md

```diff
@@ -96,7 +69,46 @@ semgrep:
 
 </p>
 </details>
-</br>
+
```

Comment:
```suggestion
<br />
```

Reply:
Does a line break do anything before a section header?

Reply:
It added a space on my local checkout of the repo.

Reply:
Note the “Standalone providers” section:

Without line break:
<img width="514" alt="Screen Shot 2020-11-04 at 2 46 43 PM" src="https://user-images.githubusercontent.com/302941/98176066-b08a6480-1eac-11eb-8461-18b7dc7a5159.png">

---

With line break:
<img width="557" alt="Screen Shot 2020-11-04 at 2 47 19 PM" src="https://user-images.githubusercontent.com/302941/98176087-ba13cc80-1eac-11eb-8b4f-425b1545a524.png">